gui/widget.cpp

```diff
@@ -305,14 +306,7 @@ StaticTextWidget::StaticTextWidget(GuiObject *boss, const Common::String &name,
 	_type = kStaticTextWidget;
 	_label = text;
 
-	_align = g_gui.xmlEval()->getWidgetTextHAlign(name);
-	if (g_gui.useRTL() && _useRTL) {
-		if (_align == Graphics::kTextAlignLeft) {
-			_align = Graphics::kTextAlignRight;
-		} else if (_align == Graphics::kTextAlignRight) {
-			_align = Graphics::kTextAlignLeft;
-		}
-	}
```

Comment:
this was 1 step towards more clean code, but not designed for the whole system. since the new abstraction is now in place, this code is useless, so remove it